README.md

```diff
@@ -46,7 +46,7 @@ Sending an email message:
 
 #### Parameters
 
-* `to`: _(mandatory)_ A list of recipients as strings.
+* `to|to_file`: _(mandatory)_ Use `to` to specify a list of recipients as strings or use `to_file` to cpecify file with addresses, each address should be in a new line.
```

Comment:
Hi @hryamzik thank you for your pull-request and sorry for the late reply. Would you mind to fix the typo "cpecify" -> "specify" and I'll gladly merge it!